salesagility/SuiteCRM
# Line breaks between words on info box

## Description

help box displays a line breaks in the middle of a sentence. It could work on english but would behave differently on other languages. 
A line break should not be between words or it will break lines differently in each language

## Motivation and Context

Bad looking info box caused by line breaks.

## How To Test This

Go to: Administrator menu  / Admin / System settings / System definitions / Import - Maximum Number of Rows: Information (click the info icon to see the box)
- This is how it looks (SuiteCRM 7.7.8 same error on both templates)
![import_information](https://cloud.githubusercontent.com/assets/3055443/21360930/dd299602-c6d9-11e6-8c8a-ae83a665c2e3.png)


- After language file edited (and a rebuild)... this is how it will look:
![2016-12-20 17_27_56-microsoft edge](https://cloud.githubusercontent.com/assets/3055443/21360907/c4e88490-c6d9-11e6-8426-8c07d3b90c3c.png)

Nice!

## Types of changes


### Final checklist
